### DISCUSSION TITLE
Get wifi code from other emulators

### DISCUSSION THREAD

**Discussion by Damaj301damaj-lol** [rafaelvcaetano/melonDS-android#771] [Ideas]
I though of it out of nowhere, i think it might even not be possible but why don't u try to check dolphin-emulator's way to enable wifi(wifi does work on android :v) [Dolphin github](https://github.com/dolphin-emu/dolphin)

**Comment by strikersix23** [comment]
wifi code for emulation is often console specific even though it may seem like a good idea it often isnt because dolphin is wii, melonds is ds/dsi. Its two different hardware types for dsi/ds to work you have to enable a specific adaptor, which wii doesnt have.

But as of currently melonds (PC) master has slirp which is userspace and should be a matter of patching it to work on android as slirp is confirmed working in the likes of termux which is a app for Android that is a mini linux enviroment that has tools and other very common apps for linux if termux compiles it, since it uses thr Android ndk it should work if the lib is wrapped
